nbclient/client.py

```diff
@@ -419,7 +430,7 @@ def setup_kernel(self, **kwargs):
 
         # Can't use run_until_complete on an asynccontextmanager function :(
         if self.km is None:
-            self.start_kernel_manager()
+            self.km = self.start_kernel_manager()
```

Comment:
Not really necessary to do the double assignment since `self.start_kernel_manager()` assigns self.km regardless, but doesn't really hurt anything either for protecting subclasses not implementing the same side-effect.

Reply:
If we don't assign to `self.km` we get `error: Item "None" of "Optional[Any]" has no attribute "has_kernel"` with the following line.